core/java/iesi-core/src/main/java/io/metadew/iesi/script/action/sql/SqlExecuteQuery.java

```diff
@@ -117,14 +117,12 @@ protected boolean executeAction() throws SQLException, InterruptedException {
         String outputDatasetReferenceName = convertDatasetReferenceName(getSqlQuery().getValue());
         boolean appendOutput = convertAppendOutput(getConnectionName().getValue());
         // Get Connection
-        Connection connection = ConnectionConfiguration.getInstance()
+        Connection connection;
+        connection = ConnectionConfiguration.getInstance()
                 .get(new ConnectionKey(connectionName, this.getExecutionControl().getEnvName()))
-                .get();
+                .orElseThrow(() -> new RuntimeException("Unknown connection name"));
```

Comment:
add 'connectionName' to the Exception message: cannot find connection 'connectionName'. This will help users during debugging

Reply:
done in next commit